README.md

```diff
@@ -49,11 +51,28 @@ The config files will be saved to the `config` directory in the repo. You can ed
 
 To run, you'll need to install the requirements using `pip install -r code/requirements.txt` and then run the script using `python3 code/aniplanrr.py`
 
+## <a name="env"></a>Env Options
+
+```
+ANILIST_USERNAME="yourusername"             # AniList Username - Mandatory
+SONARRURL="http://192.168.1.1:8989/"        # Radarr URL - Optional
+SONARRAPIKEY="yourapikey"                   # Radarr API Key - Needed if using Sonarr
+RADARRURL="http://192.168.1.1:7878/"        # Sonarr URL - Optional
+RADARRAPIKEY="yourapikey"                   # Sonarr API Key - Needed if using Radarr
+MONITOR='all'                               # Monitor Type (all, future, missing, existing, firstSeason, latestSeason, pilot) ALL IS RECOMMENDED, OTHER FLAGS MAY BE BROKEN
+RETRY=True                                  # If True, will write failed shows to ignore file to ignore next time. If not set, will not ignore anything. If set to 'manual', will ignore entries in ignore, but not add more.
+INTERVAL=3600                               # Interval in seconds to run the script on (this will run it every hour) - Necessary in containers
+LOGGING=False                               # If True, will add extra output for debug purposes! Also generates a logging folder (config/log)
+RESPECTFUL_ADDING                           # If True, will not even touch a series if it's already listed in the application
+AUTO_FILL_MAPPING                           # Allow the program to write mapping entries - See mapping down below
+```
+[Mapping](#mapping)
```

Comment:
I'd prefer if we don't link to the mapping and instead just say "See mapping in the "Config Files" section